src/org/ods/usecase/LeVADocumentUseCase.groovy

```diff
@@ -90,7 +90,8 @@ class LeVADocumentUseCase extends DocGenUseCase {
 
             def repo_ = this.project.repositories.find { [it.id, it.name, it.metadata.name].contains(normComponentName) }
             if (!repo_) {
-                throw new RuntimeException("Error: unable to create ${documentType}. Could not find a repository configuration with id or name equal to '${normComponentName}' for Jira component '${component.name}' in project '${this.project.key}'.")
+                def repoNamesAndIds = this.project.repositories.collect{ [id: it.id, name: it.name] }
+                throw new RuntimeException("Error: unable to create ${documentType}. Could not find a repository configuration with id or name equal to '${normComponentName}' for Jira component '${component.name}' in project '${this.project.key}'. Please check the metatada file. In the metadata there are the following repositories configured: ${repoNamesAndIds}")
```

Comment:
@martsec must be the time in the day ;-) `metatada`. Maybe you want to be a little more precise and say `in the metadata.yml file. In this file,...`

Reply:
True